registered/validate/validators.py

```diff
@@ -437,11 +437,13 @@ def validate_routes_have_two_directions(rating):
     - 171
     - 195
     - 214
+    - 600 (Onboard Tablet Pilot Test Route)
+    - 601 (Onboard Tablet Pilot Test Route)
     - rad
     - wad
     """
     default_expected_count = 2
-    override_counts = {"171": 1, "195": 1, "214": 1, "rad": 1, "wad": 1}
+    override_counts = {"171": 1, "195": 1, "214": 1, "600":1, "601":1, "rad": 1, "wad": 1}
```

Comment:
suggestion: looks like there's a formatting issue
```suggestion
    override_counts = {"171": 1, "195": 1, "214": 1, "600": 1, "601": 1, "rad": 1, "wad": 1}
```